docs/features/thumbnails.md

```diff
@@ -19,6 +19,8 @@ permalink: /features/thumbnails
 
 Both PrusaSlicer and SuperSlicer allow configuration of thumbnails to be generated alongside your `gcode`. Each slicer has a slightly different configuration.
 
+Cura requires configuring a post processing script to create the thumbnails.  
```

Comment:
Not strictly true.
If you use the cura-octoprint plugin, if your printer is on BEFORE cura connects to it to see its "capabilites", it will see the support for UFP documents, and upload via that instead.

So this only works if you do all of the followng:
* Cura-octoprint plugin
* octoprint_compat enabled
* printer is on when you laumch cura
* You use the "upload to cura" button.

If you use a different flow, then the post-procesing script would be needed.

Reply:
Thank you, I wasn't aware of that! I just tested by disabling the post-processing actions and using the "Print with OctoPrint" button and it worked exactly as you said!

I will update the text to follow that workflow instead, as it seems to make more sense to me than using the post-processing scripts!

Reply:
Thanks, this looks great now :smile: